python/ray/worker.py

```diff
@@ -37,6 +37,11 @@
     random_string,
 )
 
+try:
+    from ray.plasma import plasma_eventloop
+except Exception:
+    pass
```

Comment:
Let's not do this and require the user to manually do `import ray.experimental.async_api` or something like that.